package.json

```diff
@@ -1,34 +1,64 @@
 {
-  "name": "hubspot-local-dev-tools",
-  "version": "1.0.0",
-  "description": "Local development tools for working with HubSpot",
+  "name": "@hubspot/cli",
+  "version": "6.2.0",
+  "description": "CLI for working with HubSpot",
+  "license": "Apache-2.0",
   "repository": "https://github.com/HubSpot/hubspot-cli",
-  "private": true,
+  "dependencies": {
+    "@hubspot/local-dev-lib": "2.0.0",
+    "@hubspot/serverless-dev-runtime": "6.1.1",
+    "@hubspot/theme-preview-dev-server": "0.0.7",
+    "@hubspot/ui-extensions-dev-server": "0.8.33",
+    "archiver": "^5.3.0",
+    "chalk": "^4.1.2",
+    "chokidar": "^3.0.1",
+    "cli-cursor": "^3.1.0",
+    "cli-progress": "^3.11.2",
+    "express": "^4.17.1",
+    "findup-sync": "^4.0.0",
+    "fs-extra": "^8.1.0",
+    "inquirer": "8.2.0",
+    "jest": "^29.5.0",
+    "js-yaml": "^4.1.0",
+    "moment": "^2.29.1",
+    "open": "^7.0.3",
+    "ora": "^4.0.3",
+    "p-queue": "^6.0.2",
+    "strip-ansi": "^5.2.0",
+    "table": "^6.6.0",
+    "tmp": "^0.2.1",
+    "update-notifier": "^5.1.0",
+    "yargs": "15.4.1"
+  },
   "devDependencies": {
-    "depcheck": "1.3.1",
+    "axios": "^1.7.2",
+    "mock-stdin": "^1.0.0",
     "eslint": "^8.56.0",
     "husky": "^1.3.1",
-    "lerna": "^6.6.1",
     "lint-staged": "^10.5.4",
     "madge": "^6.1.0",
-    "prettier": "^1.19.1",
-    "sass": "~1.29.0"
+    "prettier": "^1.19.1"
   },
-  "workspaces": [
-    "packages/*",
-    "acceptance-tests"
-  ],
   "scripts": {
-    "check-deps": "node bin/depcheck.js",
```

Comment:
Can you remind me what's happening with `depcheck`? Do we not need this command anymore?

Reply:
Still using it, but we didn't need a custom script anymore so we're just using npx (see my comment above on build.yml)